lib/apollo-federation/schema.rb

```diff
@@ -25,6 +24,14 @@ def federation_version
         @federation_version || find_inherited_value(:federation_version, '1.0')
       end
 
+      def set_imported_directives(directives)
+        @imported_directives = DEFAULT_IMPORTED_DIRECTIVES + directives
```

Comment:
```suggestion
        @imported_directives = (DEFAULT_IMPORTED_DIRECTIVES + directives).uniq
```

Reply:
how the `set_imported_directives` is supposed to be called?

Reply:
I would love to call it here: https://github.com/productboard/pb-backend/pull/19438/files#diff-032b27e96cf7c7cbe267d7639d77752ce6ca7b0afb7fa5e5f33c24990436c0a6R3
would that be possible? 🤔

Reply:
Yeah, I think so, you could have also achieved this by doing `class_attribute`. You can also test it locally by pointing  Gemfile to local path `gem 'apollo-federation', path: '/path/to/local/gem``

Reply:
thanks, rewrote it to use `class_attribute`. I will test it locally

Reply:
If you need any help, let me know, we can pair